core/codegen/src/lib.rs

```diff
@@ -97,7 +97,9 @@ mod attribute;
 mod bang;
 mod http_codegen;
 mod syn_ext;
+mod ident;
 
+use crate::ident::Ident;
```

Comment:
Where do we use this in this file?

Reply:
Nowhere. It's here so that `crate::Ident` works instead of `crate::ident::Ident` everywhere else.

Reply:
I see. Can we add `pub(crate)` to this to make it clear?